packages/engine/Source/Core/ITwin.js

```diff
@@ -0,0 +1,306 @@
+  if (!defined(ITwin.defaultAccessToken)) {
+    throw new DeveloperError("Must set ITwin.defaultAccessToken first");
+  }
+
+  const headers = {
+    Authorization: ITwin.defaultAccessToken,
+    Accept: "application/vnd.bentley.itwin-platform.v1+json",
+  };
+
+  // obtain export for specified export id
+  const url = `${ITwin.apiEndpoint}mesh-export/${exportId}`;
+
+  // TODO: this request is _really_ slow, like 7 whole second alone for me
+  // Arun said this was kinda normal but to keep track of the `x-correlation-id` of any that take EXTRA long
+  const response = await fetch(url, { headers });
+  if (!response.ok) {
+    const result = await response.json();
+    if (response.status === 401) {
+      throw new RuntimeError(
+        `Unauthorized, bad token, wrong scopes or headers bad. ${result.error.details[0].code}`,
+      );
+    } else if (response.status === 404) {
+      throw new RuntimeError(`Requested export is not available ${exportId}`);
+    } else if (response.status === 429) {
+      throw new RuntimeError("Too many requests");
+    }
+    throw new RuntimeError(`Unknown request failure ${response.status}`);
+  }
+
+  /** @type {ExportResponse} */
+  const result = await response.json();
+  return result;
+};
+
+/**
+ * Get the list of exports for the given iModel + changeset
+ *
+ * @param {string} iModelId
+ * @param {string} changesetId
+ */
+ITwin.getExports = async function (iModelId, changesetId) {
+  if (!defined(ITwin.defaultAccessToken)) {
+    throw new DeveloperError("Must set ITwin.defaultAccessToken first");
+  }
+
+  const headers = {
+    Authorization: ITwin.defaultAccessToken,
+    Accept: "application/vnd.bentley.itwin-platform.v1+json",
+    Prefer: "return=representation", // or return=minimal (the default)
+  };
+
+  // obtain export for specified export id
+  let url = `${ITwin.apiEndpoint}mesh-export/?iModelId=${iModelId}`;
+  if (defined(changesetId) && changesetId !== "") {
+    url += `&changesetId=${changesetId}`;
+  }
+
+  const response = await fetch(url, { headers });
+  if (!response.ok) {
+    const result = await response.json();
+    if (response.status === 401) {
+      throw new RuntimeError(
+        `Unauthorized, bad token, wrong scopes or headers bad. ${result.error.details[0].code}`,
+      );
+    } else if (response.status === 422) {
+      throw new RuntimeError(
+        `Unprocessable Entity:${result.error.code} ${result.error.message}`,
+      );
+    } else if (response.status === 429) {
+      throw new RuntimeError("Too many requests");
+    }
+    throw new RuntimeError(`Unknown request failure ${response.status}`);
+  }
+
+  /** @type {{exports: Export[]}} */
+  const result = await response.json();
+  return result;
+};
+
+/**
+ * Start the export process for the given iModel + changeset.
+ *
+ * @param {string} iModelId
+ * @param {string} changesetId
+ */
+ITwin.createExportForModelId = async function (iModelId, changesetId) {
```

Comment:
Today, we're auto-generating tiles for all iModels but for the exportType=IMODEL.
We plan to also auto-generate "3DTILES". Development is on-going, we expect to have it end of this quarter.
We also plan to deprecate this endpoint when the auto-generation is in place.

Reply:
@danieliborra We plan on marking all of these new APIs as "experimental" so we have some more flexibility with breaking changes. So we can remove this function when recommended.